src/client/_clientwindow.py

```diff
@@ -1325,16 +1325,17 @@ def handle_mod_info(self, message):
         self.modInfo.emit(message)
 
     def handle_game_info(self, message):
-        if message['uid'] == self.game_session.game_uid:
-            self.game_session.game_map = message['mapname']
-            self.game_session.game_mod = message['featured_mod']
-            self.game_session.game_name = message['title']
-            self.game_session.game_visibility = message['visibility']
-
         if 'games' in message:
             for game in message['games']:
                 self.gameInfo.emit(game)
         else:
+            # sometimes we get the game_info message before a game session was created
+            if self.game_session and message['uid'] == self.game_session.game_uid:
```

Comment:
Along with that; we're not actually handling the case where updates to our own game is part of a multi-game broadcast.

@ChrisKitching this is probably something you're dealing with in the pyqt5 branch, so I suggest we just keep that in mind for there and leave this as is.